internal/api/ext_headers.h

```diff
@@ -264,6 +264,7 @@ static const int errno_private = 0;
 #include <fcntl.h>
 #include <inttypes.h>
 #include <limits.h>
+#include <stdbool.h>
```

Comment:
Shouldn’t this also be added to the non-Windows environment further down?

Reply:
Fixed